ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/JlineSimpleCompleter.java

```diff
@@ -46,17 +46,25 @@
  * @since 2.0.0
  */
 public class JlineSimpleCompleter implements Completer {
-    private final StringsCompleter topicsCompleter;
     private final StringsCompleter topicsOptionCompleter;
     private final StringsCompleter commandsCompleter;
     private final StringsCompleter keywordsCompleter;
     private final Completers.FileNameCompleter fileNameCompleter;
+    private final StringsCompleter topicsCompleter;
 
     public JlineSimpleCompleter() {
-        List<String> topicsKeywords = FileUtils.readKeywords(PropertiesLoader.getProperty(TOPICS_FILE));
+
+        BbeCompletionProvider bbeHelpProvider = new BbeCompletionProvider();
+        List<String> topicsKeywords;
+        topicsKeywords = bbeHelpProvider.getTopicList();
+        if (topicsKeywords.size() > 0) {
+            this.topicsCompleter = new StringsCompleter(topicsKeywords);
+        } else {
+            this.topicsCompleter = null;
```

Comment:
Why make this null?

Reply:
Removed null assignment in 7d6487651eae6efac7995ab50e34671adc01c785